lib/developer_portal/app/views/developer_portal/main_layout.html.liquid

```diff
@@ -14,9 +14,9 @@
 
     {% csrf %}
     {{ '/css/bootstrap.css' | stylesheet_link_tag }}
-    {{ '//netdna.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.css' | stylesheet_link_tag }}
+    {% cdn_asset /font-awesome/4.3.0/css/font-awesome.css %}
```

Comment:
So we would need to document those for customers to know what assets we are hosting

Documentation should go to https://github.com/3scale/porta/blob/0b98c4d6620a6fea2f55d2fff64c674fb9764452/lib/developer_portal/lib/liquid/tags/cdn_asset.rb#L8


Provide in the description:

```
The assets you can use here are limited to:

- font-awesome/4.3.0/css/font-awesome.css
- font-awesome/4.3.0/css/font-awesome.min.css
- jquery-ui/1.11.4/jquery-ui.css 
- jquery-ui/1.11.4/jquery-ui.min.js
- jquery/1.7.1/jquery.min.js
etc ....
```

Reply:
Done [here](https://github.com/3scale/porta/pull/1468/files#diff-1030a186b26a17a9e79dd0d24e0178d5R19) @hallelujah